### PR TITLE
Nav component

### LINKED ISSUE #10: Navigation
Even after having same Navigation bar Each page has its  own individual code which is creating duplication .

### PR DESCRIPTION
**Pull request by tanishk15sharma**
Fixes #10 

- Created `nav.js` file
- Created `nav.css` file 
- Removed nav code from each page
- Removed nav css from each page

### CODE REVIEW

**Review by prathmeshjagtap — APPROVED** [review]
It's good that you have made a separate component for nav rather than repeating the code everywhere 

I like that you are using calc property  for nav to calculate depending on screen size

File: src/pages/ProductsPage/ProductsPage.css [hunk 1]
```diff
@@ -23,45 +23,9 @@ main {
     margin: 1rem;
   }
 
-  .foot-icon {
-    margin: 0;
-  }
-  nav {
-    /* border: 1px solid red; */
```
Comment (deepak29-git) [inline src/pages/ProductsPage/ProductsPage.css:30]: remove commented code

**Review by deepak29-git — APPROVED** [review]
your folder structure is good .

File: src/components/homepage/Header.js [hunk 2]
```diff
@@ -1,14 +1,11 @@
 import "./Header.css";
-import { useState } from "react";
-import { Link } from "react-router-dom";
-import searchIcon from "../../assets/nav-icons/search.svg";
-import likeIcon from "../../assets/nav-icons/heart.svg";
-import loginIcon from "../../assets/nav-icons/user-plus.svg";
-import cartIcon from "../../assets/nav-icons/shopping-cart.svg";
+// import { useState } from "react";
```
Comment (Mohitkumar33) [inline src/components/homepage/Header.js:2]: you can remove commented code

File: src/components/homepage/Header.js [hunk 3]
```diff
@@ -1,14 +1,11 @@
...
+import { Nav } from "../../components/nav/nav";
+
 import { useCart } from "../../contexts/cart-context";
 
 const HeaderSection = () => {
-  const [toggleHamburger, setToggleHamburger] = useState(false);
+  // const [toggleHamburger, setToggleHamburger] = useState(false);
```
Comment (Mohitkumar33) [inline src/components/homepage/Header.js:8]: here also

**Review by Mohitkumar33 — APPROVED** [review]
Nice folder structure

**Comment by tanishk15sharma** [comment]
> It's good that you have made a separate component for nav rather than repeating the code everywhere
> 
> I like that you are using calc property for nav to calculate depending on screen size

yes , thank you

File: src/pages/ProductsPage/ProductsPage.css (hunk 1 above)
Comment (tanishk15sharma) [reply src/pages/ProductsPage/ProductsPage.css:30]: okay

**Pull request merged**